Source/Project64/User Interface/Frame Per Second Class.cpp

```diff
@@ -49,16 +49,17 @@ void CFramePerSecond::Reset (bool ClearDisplay)
 		g_Notify->DisplayMessage2(L"");
 		return;
 	}
-	
-	if (m_iFrameRateType == FR_VIs) 
-	{
-		DisplayViCounter(0);
-	}
+
+	if (m_iFrameRateType == FR_VIs)

```

Comment:
Display DisplayViCounter is never being called now

I assume you meant if (m_iFrameRateType == FR_VIs || m_iFrameRateType == FR_VIs_DLs)